pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -181889,7 +181889,7 @@ self: {
        testHaskellDepends = [ base bytestring hspec ];
        description = "A low-level MySQL client library";
        license = lib.licenses.bsd3;
-     }) {inherit (pkgs) mysql;};
+     }) {};
```

Comment:
This file is generated automatically, so this will just be rolled back.

Adressing this involves fixing <https://github.com/NixOS/cabal2nix/issues/500>, updating `cabal2nix-unstable` in nixpkgs and regenerating the package set which I'll try to get to soon.

I'm a bit surprised that this is even re-generated by `hackage2nix` as we use `allowAliases = false;` when searching for attribute names.

Reply:
I see, I'll remove the change then. It actually seems to build fine without it anyway, so fixing it isn't urgent.